libs/utils/utils.js

```diff
@@ -865,8 +865,9 @@ export async function loadMartech({
     return true;
   }
 
-  const query = PAGE_URL.searchParams.get('martech');
-  if (query === 'off' || getMetadata('martech') === 'off') {
+  const martechQuery = PAGE_URL.searchParams.get('martech');
+  const marketingQuery = PAGE_URL.searchParams.get('marketingtech');
+  if (martechQuery === 'off' || marketingQuery === 'off' || getMetadata('martech') === 'off') {
```

Comment:
Nit: you can simplify this by not caching
```suggestion
  if (PAGE_URL.searchParams.get('martech') === 'off' 
  || PAGE_URL.searchParams.get('marketingtech') === 'off' 
  || getMetadata('martech') === 'off') {
```

Reply:
sounds good.  added your suggestion